07-regression1.Rmd

```diff
@@ -301,7 +315,8 @@ Here we can see that the smallest RMSPE occurs when $K =$ `r sacr_min %>% pull(n
 
 ## Underfitting and overfitting
 Similar to the setting of classification, by setting the number of neighbours
-to be too small or too large, we cause the RMSPE to increase:
+to be too small or too large, we cause the RMSPE to increase, as shown in
+Figure \@ref(fig:07-choose-k-knn-plot).  What is happening here? 
 
 ```{r 07-choose-k-knn-plot, echo = FALSE, fig.height = 4, fig.width = 5, fig.cap = "Effect of the number of neighbours on the RMSPE"} 
```

Comment:
So I think this plot belongs above, and then we can refer to it again here in this section.

Reply:
@ttimbers not sure what you mean

Reply:
oh nevermind, it links with the above comments. got it

Reply:
handled